include/mercury/http/http_client.h

```diff
@@ -21,17 +21,19 @@
 
 #include "mercury/http/http_client_interface.h"
 
-
 namespace Mercury {
 namespace HTTP {
 
+#define DEFAULT_HTTP_CLIENT_TIMEOUT    15 // seconds
+
 /**
  * \class HTTPClient
  * \brief Simple implementation of IHTTPClient
  */
 class HTTPClient : public IHTTPClient {
  public:
     HTTPClient();
+    HTTPClient(int timeout_secs);
```

Comment:
Combine these two constructors into one?

```suggestion
    HTTPClient(int timeout_secs = DEFAULT_HTTP_CLIENT_TIMEOUT);
```